mappings/net/minecraft/world/gen/stateprovider/RandomizedIntBlockStateProvider.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_5929 net/minecraft/world/gen/stateprovider/RandomizedIntBlockStateProvider
+	COMMENT A {@linkplain BlockStateProvider block state provider} that randomizes a single {@link net.minecraft.state.property.IntProperty} of a block state provided by another provider.
```

Comment:
```suggestion
	COMMENT A {@linkplain BlockStateProvider block state provider} that randomizes a single {@link IntProperty} of a block state provided by another provider.
```
no need for qualification as it's already used in fields

Reply:
Fixed in 12bc1e25c87248206e10b07df41b2576097cd52d